src/GHC.hs

```diff
@@ -29,7 +29,7 @@ defaultKnownPackages =
     , filepath, genapply, genprimopcode, ghc, ghcBoot, ghcBootTh, ghcCabal, ghci
     , ghcPkg, ghcPrim, ghcTags, haddock, haskeline, hsc2hs, hoopl, hp2ps, hpc
     , hpcBin, integerGmp, integerSimple, iservBin, libffi, mkUserGuidePart
-    , parallel, pretty, primitive, process, rts, runGhc, stm, templateHaskell
+    , pretty, process, rts, runGhc, templateHaskell
```

Comment:
@snowleopard I have rebased this PR once again so it is based on your latest master. With the changes you made in 67f433bf028ec4c4251b928fa476ff1302e8299c _hadrian_ can build _ghc_ from the created tarball. I think my work on this PR is done.